pyaedt/generic/DataHandlers.py

```diff
@@ -482,6 +489,36 @@ def to_aedt(code):
     return return_code
 
 
+def str_to_bool(s):
```

Comment:
@dcrawforAtAnsys I'd make it more general. 
The input will always be a string but we can check whether this string is in a list of specific value to check if it's True (i.e ["true", "yes", "y", 1 etc.] or False (["false", "no", "0" etc.]). If the type doesn't comply with string then we return the string in itself as you are already doing.

Reply:
@gmalinve I will add your suggestion and document it.

Reply:
@gmalinve : I will do this and update the dosctring.